packages/graph/hash_graph/lib/graph/src/knowledge/mod.rs

```diff
@@ -36,7 +36,9 @@ pub struct EntityQuery {
     #[component(value_type = number)]
     pub entity_type_query_depth: QueryDepth,
     #[component(value_type = number)]
-    pub linked_entity_query_depth: QueryDepth,
+    pub link_target_entity_query_depth: QueryDepth,
```

Comment:
I missed this before but do the `QueryDepth` docs still only talk about ontology elements? We might want to either make a separate alias or fix the docs

Reply:
I think I will add a new alias, in the case we change the type, it's likely that it's only changed for either the ontology or the knowledge

Reply:
Added in https://github.com/hashintel/hash/pull/1074/commits/c92b8910eb27c5dcedb1fad949ef8fbdc4f2b0eb, also adjusted the documentation to match the behavior. Note: it's the same behavior, but entities and links are chained differently from ontology types. I struggled a bit with the documentation, feel free to adjust it 🙂